internal/fuzzbinder/fuzzscraper.go

```diff
@@ -424,11 +318,12 @@ func GetFuzzTargets(fuzzParameters *FuzzParameters) ([]string, error) {
 }
 
 // addClaimEvidence adds an evidence to the list of the evidence files used by the fuzzscraper.
-func addClaimEvidence(evidences []amber.ClaimEvidence, blobName string, role string) ([]amber.ClaimEvidence, error) {
-	digest, err := getGCSFileDigest(CoverageBucket, blobName)
+func addClaimEvidence(ctx context.Context, client *gcsutil.GCSClient, evidences []amber.ClaimEvidence, blobName string, role string) ([]amber.ClaimEvidence, error) {
+	fileBytes, err := client.GetBlobData(ctx, CoverageBucket, blobName)
 	if err != nil {
-		return nil, fmt.Errorf("could not get digest of evidence %s: %v", blobName, err)
+		return nil, fmt.Errorf("could not get date in evidence file: %v", err)
```

Comment:
Does this error make sense? do we try to get the data in evidence file here?

Reply:
Oh, it's `data`, I fixed it. Thanks !